sample/commons.gradle

```diff
@@ -38,7 +38,7 @@ ext.getBaseVersionCode = { version ->
 }
 
 ext.getBuildNumber = {
-    def final build
```

Comment:
I had to change this because of: 
GestureViews/sample/commons.gradle': 48: The variable [build] is declared final but is reassigned
  . At [48:9]  @ line 48, column 9.
             build = 1